bravado/fido_client.py

```diff
@@ -1,19 +1,15 @@
 # -*- coding: utf-8 -*-
 import logging
-
+import requests
```

Comment:
Please retain the blank line between standard library imports and 3rd party imports.